### ISSUE TITLE
Adding Export Key functionality in Web Inspector

### ISSUE THREAD

**Issue by kkanag314** [Metrological/WebKitForWayland#725]
Requesting the inclusion of export functionality in the Web Inspector tool in wpe_2.22.
SRCREV = "22d6522031250cdeb967101a0fa0cc91f190b84d"
The export key under network tab will be used to download  the .har file through webinspector.
There is one option called "Export" under the network tab. But the functionality is not enabled.

**Comment by magomez** [comment]
I've just pushed 405e52d3e520fb00caa9f1950d3eeebcbda7c926 to the wpe-2.22 branch to fix this. Clicking on the export button should download a file with the info. Please give it a test and close this if it works for you.

**Comment by magomez** [comment]
Ah, I forgot to mention that the export feature doesn't work if there's no content in the network log (the export button will be disabled). To make it work, with the inspector network tab loaded, do some navigation on the inspected page (or reload it). This will start filling the network log and the export option will be enabled.

**Comment by kkanag314** [comment]
@magomez I cherry-picked the changes in wpe_webkit_2.22.But the Export key functionality is still disabled. I/m not able to download the HAR file using Export key under network tab.

![image](https://user-images.githubusercontent.com/62739415/138679299-40664c1d-5a4d-4347-accc-120c3ae5ba51.png)

**Comment by magomez** [comment]
@kkanag314 please see my previous comment. The export button is disabled while the content log is empty, as there's nothing to export. In the image you attached it's empty (despite you see those file names, the transfer info is empty). Please, with that tab open, do some navigation on the inspected page or reload the page. That will fill the log and the export option will become enabled.

**Comment by kkanag314** [comment]
@magomez  I tried to do some navigation/played contents in the webapp and reload the page. But the export key is still disabled.

![image](https://user-images.githubusercontent.com/62739415/138688635-9a37589b-4599-4eac-bf8f-72a5313c38ce.png)

**Comment by magomez** [comment]
That's weird. Clicking on the button should download the file. What do you mean with the option being disabled? The button doesn't react to the mouse over, like the other buttons on the same row (All, Document, CSS, etc), but it should work when clicked.

Which browser are you using? I've tested on firefox (it opens a save as dialog") and chrome (it directly downloads the file). Maybe you have popups disabled by default? As it's like clicking on a link, the browser might be blocking the download. Check whether there are warnings that windows have been blocked.

Also, the change I made will change libWPEWebInspectorResources.so. so be sure that this library is reinstalled after wpe is rebuilt with my change.

Can you test those things and tell me whether they fix the problem?

**Comment by kkanag314** [comment]
@magomez The libWPEWebInspectorResources.so is converted as a downloadable module in our embedded devices .So When I flashed the image, the corresponding webinspector library is not downloaded due to some server error.Due to this It was using the existing  WPEWebInspector library downloaded in the device.
I tried with the libWPEWebInspectorResources.so which has this changes. I'm able to download the HAR files using Export Key. Thank you
